sql-plugin/src/main/scala/com/nvidia/spark/rapids/FloatUtils.scala

```diff
@@ -18,6 +18,8 @@ package com.nvidia.spark.rapids
 
 import ai.rapids.cudf.{ColumnVector, ColumnView, DType, Scalar}
 
+import org.apache.spark.sql.types.DecimalType
```

Comment:
I don't think this is used any more and some code was just merged in to check for unused imports and report an error.